dao/mongodb.rya/src/main/java/org/apache/rya/mongodb/MongoDBRdfConfiguration.java

```diff
@@ -197,22 +195,22 @@ public void setMongoPassword(final String password) {
      * @return The name of the Rya instance to connect to. (default: rya)
      */
     public String getRyaInstanceName() {
-        return get(MONGO_COLLECTION_PREFIX, "rya");
+        return get(MONGO_DB_NAME, "rya");
```

Comment:
So an application can say set 

setMongoDBName("a")
setRyaInstanceName("b")

And then getMongoDBName() returns "b"? That's unexpected.

Reply:
yeah that's a good point